sdcm/fill_db_data.py

```diff
@@ -2891,12 +2893,17 @@ def cql_create_tables(self, session):
         for truncate in truncates:
             session.execute(truncate)
 
+    @retrying(n=3, sleep_time=20, allowed_exceptions=ProtocolException)
+    def truncate_table(self, session, truncate):  # pylint: disable=no-self-use
+        LOGGER.debug(truncate)
+        session.execute(truncate)
+
     def truncate_tables(self, session):
         # Run through the list of items and create all tables
```

Comment:
why it is written "create all tables" ? isn't it only truncates tables?

Reply:
You are right, I will fix the comment later